include/graft_manager.h

```diff
@@ -272,6 +272,7 @@ class GraftServer final
 
 private:
     static void ev_handler(mg_connection *client, int ev, void *ev_data);
+    int methodFromString(std::string& method);
```

Comment:
Better to have `method` parameter as const reference, as it input-only parameter?